### PR TITLE
Expanded English.json with more quotes

### PR DESCRIPTION
**Pull request by holyhearted**
11 more (longer) quotes to english.json

### CODE REVIEW

**Comment by Miodec** [comment]
Please go through all the quotes and remove any that are not PG13

**Review by Miodec — CHANGES_REQUESTED** [review]
Please remove any God references. People get really worked up about this and create drama for a dumb reason so I just wanna avoid the topic all together. Also any political quotes too

**Comment by holyhearted** [comment]
> Please remove any God references. People get really worked up about this and create drama for a dumb reason so I just wanna avoid the topic all together. Also any political quotes too

i believe you’re referencing an older version of this file, all commented lines have been removed and replaced (have they not?)

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -35142,6 +35142,180 @@
...
+      "id": 6164
+    },
+    {
+      "text": "Last night I dreamt I went to Manderley again. It seemed to me I stood by the iron gate leading to the drive, and for a while I could not enter for the way was barred to me. There was a padlock and a chain upon the gate. I called in my dream to the lodge-keeper, and had no answer, and peering closer through the rusted spokes of the gate I saw that the lodge was uninhabited. No smoke came from the chimney, and the little lattice windows gaped forlorn.",
+      "source": "Rebecca - Daphne du Maurier",
+      "length": 454,
+      "id": 6165
+    },
+    {
+      "text": "Anyone can change the direction of his life, any time at all, if only he has enough motivation: that was the moral of the story. The cat found it easier to believe this than to think about what it actually meant: that the word anyone actually referred to a very small group of people, that time has no direction, and that motivation is rarely the salient difference between people.",
```
Comment (holyhearted) [inline frontend/static/quotes/english.json:35189]: remove (?)

File: frontend/static/quotes/english.json [hunk 2]
```diff
@@ -35142,6 +35142,180 @@
...
+      "id": 6168
+    },
+    {
+      "text": "I asked you, \"How is your sister? I heard she got her degree.\" And I said, \"That makes me feel old.\" You said, \"What does that make me?\"",
+      "source": "Phoebe Bridgers - Scott Street",
+      "length": 136,
+      "id": 6169
+    },
+    {
+      "text": "You were screaming at the Evangelicals - they were screaming right back, from what I remember - when you said, \"I will never be your vegetable.\" Because I think when you're gone it's forever.",
```
Comment (holyhearted) [inline frontend/static/quotes/english.json:35213]: to remove

File: frontend/static/quotes/english.json [hunk 3]
```diff
@@ -35142,6 +35142,180 @@
       "source": "Someone Who Will Love You in All Your Damaged Glory - Raphael Bob-Waksberg",
       "length": 207,
       "id": 6158
+    },
+    {
+      "text": "And if I breathe you, will it kill me? Will you have me, or watch me fall? If I fix you, will you hate me? And would you fuck this, and watch us fall?",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:35147]: Remove quotes with swear words

**Review by Miodec — CHANGES_REQUESTED** [review]

**Review by Miodec — APPROVED** [review]

**Pull request merged**